src/Vectorface/SnappyRouter/Request/HttpRequest.php

```diff
@@ -145,6 +167,42 @@ public function setPost($postData)
     }
 
     /**
+     * Returns the input stream data for the current request
+     * @return string The input stream data
+     */
+    public function getInputStream()
```

Comment:
Since you're returning a string rather than a stream, I would rename this function to simply `getInput`, `getContent`, `getBody`, or similar.

You could optionally re-factor some of this out into a `getInputStream` method that returns a stream resource, and another method to fetch the request body (if any).